scripts/scil_surface_apply_transform.py

```diff
@@ -68,23 +71,23 @@ def main():
     args = parser.parse_args()
     logging.getLogger().setLevel(logging.getLevelName(args.verbose))
 
-    assert_inputs_exist(parser, [args.in_surface, args.ants_affine],
-                        args.ants_warp)
+    assert_inputs_exist(parser, [args.in_moving_surface, args.in_transfo],
+                        args.in_deformation)
     assert_outputs_exist(parser, args, args.out_surface)
 
     # Load mesh
-    mesh = load_mesh_from_file(args.in_surface)
+    mesh = load_mesh_from_file(args.in_moving_surface)
 
-    warp_img = None
-    # Load affine
-    affine = np.loadtxt(args.ants_affine)
+    # Load transformation
+    transfo = load_matrix_in_any_format(args.in_transfo)
 
-    # Non linear transformation
-    if args.ants_warp:
-        # Load warp
-        warp_img = nib.load(args.ants_warp)
+    deformation_data = None
+    if args.in_deformation is not None:
+        deformation_data = np.squeeze(nib.load(
+            args.in_deformation).get_fdata(dtype=np.float32))
 
-    mesh = apply_transform(mesh, affine, warp_img)
+    mesh = apply_transform(mesh, transfo, deformation_data,
```

Comment:
In that function, if I dont use --inverse I get

UnboundLocalError: local variable 'inv_affine' referenced before assignment